packages/pluggableWidgets/timeline-web/package.json

```diff
@@ -18,7 +18,7 @@
     "lint": "..\"/../../node_modules/.bin/eslint\" --config ../../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
     "test": "pluggable-widgets-tools test:unit:web",
     "pretest:e2e": "node ../../../scripts/test/updateAtlas.js",
-    "test:e2e": "pluggable-widgets-tools test:e2e:web --mx-version 8",
+    "test:e2e": "pluggable-widgets-tools test:e2e:web",
```

Comment:
This feels unrelated to the scope of the PR 😄

Reply:
Rebase problem apparently